### ISSUE TITLE
How to ignore all post with error?

### ISSUE THREAD

**Issue by ferrada01** [rarcega/instagram-scraper#427]
Repeated error httperror('429 Client error: - for url: https://www.instagr....)
How to set i(ignore) to all failed post?

thank! (:

**Comment by VasiliPupkin256** [comment]
It is a temporal request limit error for your ip and/or username, it can't be ignored

**Comment by ferrada01** [comment]
Do you know how to set the answer "(A)bort (I)gnore (R)etry  or retry (F)orever"  like Ignore as default?

**Comment by VasiliPupkin256** [comment]
Ignore wouldn't solve 429 Client error it is a temporal ban it will only be lifted after some time

**Comment by ferrada01** [comment]
if i don't want to solve the 429 error, but i want to set as default (i)gnore that error by default, do you know how to do that?

**Comment by VasiliPupkin256** [comment]
There is no such option for now. If there are repeated systematic errors they should be fixed not ignored. This option can also lead to poor bug reporting, users wouldn't check logs and errors will be left unnoticed.

**Comment by redijedi** [comment]
> There is no such option for now. If there are repeated systematic errors they should be fixed not ignored. This option can also lead to poor bug reporting, users wouldn't check logs and errors will be left unnoticed.

The tool accepts a Try Forever flag. So there is already precedent for automatically attempting to move forward by simply retrying forever. But more importantly, given many (I assume I'm not the only one) run this tool non-interactively on AWS/other where cycles cost money, I think it would be preferable to either log and skip errors (even if that means effectively ending script execution), or log and error out. This would reduce my costs and bring the problem to my attention.

**Comment by VasiliPupkin256** [comment]
> I think it would be preferable to either log and skip errors (even if that means effectively ending script execution), or log and error out.

But this is what it does by default if you don't pass any option. It retries a few times and then just reports error and stops.

**Comment by redijedi** [comment]
Interesting. I see that when I attempt an authenticated scrape it hung on 429 errors waiting for input. Unauthenticated scrapes do appear to error out on certain conditions do to a missing local file (unrelated bug or misuse I think), but I haven't seen a failed http response from unauthenticated calls to verify like for like.

**Comment by VasiliPupkin256** [comment]
Yes it stops and wait for input. It doesn't terminate but it doesn't eat resources.
I have a strange feeling like there was implemented a switch to disable prompts, but I don't see it know. Hm...

**Comment by redijedi** [comment]
I hope for a disable prompts option. It may not eat resources, but keeping a process alive keeps the container/job/other alive. That costs money. Likely less due to decreased resource usage, but money still. Thanks for looking into this!

**Comment by VasiliPupkin256** [comment]
I've found the discussion here #163 it wasn't implemented because the issue causing it was fixed elsewhere
